pkg/operator/sync.go

```diff
@@ -126,6 +142,76 @@ func (optr *Operator) syncTerminationHandler(config *OperatorConfig) error {
 	return nil
 }
 
+func (optr *Operator) syncWebhookConfiguration(config *OperatorConfig) error {
+	if err := optr.syncValidatingWebhook(); err != nil {
+		return err
+	}
+
+	return optr.syncMutatingWebhook()
+}
+
+func (optr *Operator) syncValidatingWebhook() error {
+	client := optr.kubeClient.AdmissionregistrationV1().ValidatingWebhookConfigurations()
+	requiredHook := mapiv1.NewValidatingWebhookConfiguration()
+	getHook, err := optr.validatingWebhookLister.Get(requiredHook.Name)
```

Comment:
What about calling this `currentHook`, it's a bit more obvious what it is I think. To me, `getHook` is a function name, because it sounds like an action